sql/parser/create.go

```diff
@@ -276,7 +279,10 @@ func (node *ColumnTableDef) Format(buf *bytes.Buffer, f FmtFlags) {
 			buf.WriteByte(')')
 		}
 	}
-
+	if node.Family != "" {
```

Comment:
`CONSTRAINT fam FAMILY foo` doesn't currently round-trip since you're discarding the constraint name.